yarn.lock

```diff
@@ -331,9 +327,9 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
-"@github/template-parts@https://github.com/muxinc/template-parts.git#mux-player":
+"@github/template-parts@git+https://github.com/muxinc/template-parts.git#mux-player":
   version "0.4.0"
```

Comment:
This `git+https` works, but looks a bit odd.

Reply:
we're waiting on an official release of the template-parts module and have this in the meantime.